community-content/alphafold_on_workbench/Dockerfile

```diff
@@ -42,13 +31,6 @@ RUN git clone --branch v3.3.0 https://github.com/soedinglab/hh-suite.git /tmp/hh
     && popd \
     && rm -rf /tmp/hh-suite
 
-# Install Miniconda package manager.
-RUN wget -q -P /tmp \
-  https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
-    && bash /tmp/Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda \
-    && rm /tmp/Miniconda3-latest-Linux-x86_64.sh
-
-# Install conda packages.
```

Comment:
This is unnecessary, because miniconda is already installed by gcr.io/deeplearning-platform-release/base-cu110